R/guide-axis.R

```diff
@@ -41,7 +44,7 @@
 #'
 #' # can also be used to add a duplicate guide
 #' p + guides(x = guide_axis(n.dodge = 2), y.sec = guide_axis())
-guide_axis <- function(title = waiver(), check.overlap = FALSE, angle = NULL,
+guide_axis <- function(title = waiver(), check.overlap = FALSE, angle = waiver(),
```

Comment:
The default is now `waiver()` because `coord_polar2()` may overrule angle for display purposes, but `NULL` still uses the theme's angle.